src/js/interfaces/Attachment.ts

```diff
@@ -0,0 +1,24 @@
+export interface Attachment {
+  id: number;
+  contentType: string;
+  size: number;
+  name: string;
+}
+
+export interface AttachmentWithURLProps {
+  id: number;
+  contentType: string;
+  size: number;
+  name: string;
+  iso: string;
+  layerTitle: string;
+  sublayerID: number;
+  specificFeatureID: number;
+}
```

Comment:
We can use intersection type here as AttachmentWithURLProps is a combination of the two. Thoughts?
```suggestion
export type AttachmentWithURLProps = Attachment & URLProperties;
```